addons/l10n_ar_withholding/wizards/account_payment_register.py

```diff
@@ -0,0 +1,181 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import logging
+
+from odoo import models, fields, api, Command, _
+from odoo.exceptions import UserError
+
+_logger = logging.getLogger(__name__)
+
+
+class AccountPaymentRegisterWithholding(models.TransientModel):
+    _name = 'account.payment.register.withholding'
+    _description = 'Payment register withholding lines'
+
+    payment_register_id = fields.Many2one('account.payment.register', required=True, ondelete='cascade')
+    currency_id = fields.Many2one(related='payment_register_id.currency_id')
+    name = fields.Char(string='Number', required=False, default='/')
+    tax_id = fields.Many2one('account.tax', required=True)
+    l10n_ar_withholding_sequence_id = fields.Many2one(related='tax_id.l10n_ar_withholding_sequence_id')
+    base_amount = fields.Monetary(required=True, compute='_compute_base_amount', store=True, readonly=False)
+    amount = fields.Monetary(required=True, compute='_compute_amount', store=True, readonly=False)
+
+    @api.depends('tax_id', 'payment_register_id.line_ids', 'payment_register_id.amount', 'payment_register_id.currency_id')
+    def _compute_base_amount(self):
+        base_lines = self.payment_register_id.line_ids.move_id.invoice_line_ids
+        supplier_recs = self.filtered(lambda x: x.payment_register_id.partner_type == 'supplier')
+        for rec in supplier_recs:
+            old_base_amount = rec.base_amount
+            amount_total = sum(rec.payment_register_id.mapped('line_ids.move_id.amount_total'))
```

Comment:
??? (this looks dangerous)

Reply:
i Delete old_base_amount.

Reply:
@maq-adhoc is this ready to be resolved? Os is still pending?

Reply:
DONE!